package.json

```diff
@@ -1,102 +1,58 @@
 {
   "name": "sparql-benchmark-runner",
   "version": "2.9.4",
+  "packageManager": "yarn@1.22.22",
   "description": "Executes a query set against a given SPARQL endpoint",
-  "repository": "https://github.com/comunica/sparql-benchmark-runner.js",
-  "keywords": [
-    "sparql",
-    "benchmark",
-    "runner"
-  ],
   "license": "MIT",
-  "bin": {
-    "sparql-benchmark-runner": "./bin/sparql-benchmark-runner"
```

Comment:
Can we keep the `bin/sparql-benchmark-runner` file as-is? It allows better consistency between running this bin file within and outside the dev environment.